### DISCUSSION TITLE
Bitwarden Send

### DISCUSSION THREAD

**Discussion by utahbmxer** [dani-garcia/vaultwarden#1481] [Ideas]
Not super familiar with the plumbing of the new Bitwarden Send feature and how much of a lift it is, but it seems like a cool feature.  Are there plans to support this feature/api in a future version of bitwarden_rs?

Thanks!

**Comment by BlackDex** [comment]
See https://github.com/dani-garcia/bitwarden_rs/issues/246#issuecomment-796966385
